Directory.Build.props

```diff
@@ -43,10 +47,21 @@
     <LangVersion>latest</LangVersion>
     <RepositoryUrl>$(PackageProjectUrl).git</RepositoryUrl>
     <SignAssembly>true</SignAssembly>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

Comment:
I think I've raised this before, but this is redundant for this package as symbols are embedded.

In this file there should only be `SymbolPackageFormat` or

Reply:
I changed this to guard against an issue I had with publishing one of my own packages which stil used the old format where they get published from AppVeyor to a deprecated server which has since had expired TLS cert that made publishing fail: https://ci.appveyor.com/project/martincostello/sqllocaldb/builds/28012360#L181

Even if it's not in use, setting this means if we change in the future it's future-proofed.